mls-rs/src/external_client/builder.rs

```diff
@@ -477,6 +480,30 @@ pub(crate) struct Settings {
     pub(crate) cache_proposals: bool,
 }
 
+impl Debug for Settings {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("Settings")
+            .field("extension_types", &self.extension_types)
```

Comment:
I implemented `Debug` manually for (almost) any type that derived `Debug` and had a field of bytes. `ExternalClientBuilder`, part of our public API, implements `Debug`, which relies on other types including this one to implement `Debug` too. It's usually good practice to implement `Debug` for public types so that users can troubleshoot issues.